plugins/cloudtrail/pkg/cloudtrail/cloudtrail.go

```diff
@@ -48,7 +48,7 @@ const (
 	PluginName               = "cloudtrail"
 	PluginDescription        = "reads cloudtrail JSON data saved to file in the directory specified in the settings"
 	PluginContact            = "github.com/falcosecurity/plugins/"
-	PluginVersion            = "0.11.0"
+	PluginVersion            = "0.12.0"
```

Comment:
A commit is still in the preparation stage. I've added concurrency to list the keys. This speeds up the whole process if all accounts in an organisation are affected (~5min instead of ~35min for approx. 900 accounts with a time interval of 1h).

Reply:
Great!

Generally speaking, we prefer to introduce all these improvements in bulk into the 
Please let us know your plans. If you already have a list of things you want to add, I recommend opening a GitHub issue and tracking them all. It would be easier for us :pray:

Reply:
To be honest: I don't really have a plan. I'm just working on getting my use case (analyzing organization CloudTrail with tool X) to work smoothly. #429 should be the last commit.

The changes I still need concern another project/repo.